openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/explore/explore.java

```diff
@@ -0,0 +1,363 @@
+    openExplorePage();
+    Thread.sleep(1000);
+    explorePage.topics().click();
+    int topicCount = Integer.parseInt(explorePage.getTopicCount().getText());
+    int getServiceCount = 0;
+    List<WebElement> listOfItems = explorePage.serviceName();
+    List<WebElement> countOfItems = explorePage.serviceCount();
+    List<String> Names = new ArrayList<>();
+    List<Integer> count = new ArrayList<>();
+    for (WebElement sName : listOfItems) {
+      Names.add(sName.getText());
+      if (Names.contains("Tier1")) {
+        break;
+      }
+    }
+    for (int i = 0; i < Names.size() - 1; i++) {
+      count.add(Integer.parseInt(countOfItems.get(i).getText()));
+      getServiceCount += count.get(i);
+    }
+    Assert.assertEquals(getServiceCount, topicCount);
+  }
+
+  @Test
+  @Order(3)
+  public void checkDashboardCount() throws InterruptedException {
+    explorePage explorePage = new explorePage(webDriver);
+    openExplorePage();
+    explorePage.dashboard().click();
+    int dashboardCount = Integer.parseInt(explorePage.getDashboardCount().getText());
+    int getServiceCount = 0;
+    List<WebElement> listOfItems = explorePage.serviceName();
+    List<WebElement> countOfItems = explorePage.serviceCount();
+    List<String> Names = new ArrayList<>();
+    List<Integer> count = new ArrayList<>();
+    for (WebElement sName : listOfItems) {
+      Names.add(sName.getText());
+      if (Names.contains("Tier1")) {
+        break;
+      }
+    }
+    for (int i = 0; i < Names.size() - 1; i++) {
+      count.add(Integer.parseInt(countOfItems.get(i).getText()));
+      getServiceCount += count.get(i);
+    }
+    Assert.assertEquals(getServiceCount, dashboardCount);
+  }
+
+  @Test
+  public void checkPipelineCount() throws InterruptedException {
+    explorePage explorePage = new explorePage(webDriver);
+    openExplorePage();
+    Thread.sleep(1000);
+    explorePage.pipeline().click();
+    Thread.sleep(1000);
+    int pipelineCount = Integer.parseInt(explorePage.getPipelineCount().getText());
+    int getServiceCount = 0;
+    List<WebElement> listOfItems = explorePage.serviceName();
+    List<WebElement> countOfItems = explorePage.serviceCount();
+    List<String> Names = new ArrayList<>();
+    List<Integer> count = new ArrayList<>();
+    for (WebElement sName : listOfItems) {
+      Names.add(sName.getText());
+      if (Names.contains("Tier1")) {
+        break;
+      }
+    }
+    for (int i = 0; i < Names.size() - 1; i++) {
+      count.add(Integer.parseInt(countOfItems.get(i).getText()));
+      getServiceCount += count.get(i);
+    }
+    Assert.assertEquals(getServiceCount, pipelineCount);
+  }
+
+  @Test
+  public void checkBasics() throws Exception {
+    explorePage explorePage = new explorePage(webDriver);
+    tableDetails tableDetails = new tableDetails(webDriver);
+    myDataPage myDataPage = new myDataPage(webDriver);
+    openExplorePage();
+    Thread.sleep(1000);
```

Comment:
Please avoid excessive usage to `Thread.sleep()`. Rather use explicit wait in selenium

Comment applies to other instances of this in this PR